### ISSUE TITLE
Inherited attribute is repeated twice in python

### ISSUE THREAD

**Issue by vniederb** [doxygen/doxygen#11260]
**Describe the bug**
If an inherited member is accessed this member will appear twice in the html documentation.
Once as inherited member and once as a "new" member of the inheriting class

**Screenshots**
HTML Page:
![grafik](https://github.com/user-attachments/assets/4a3191ca-d457-4592-adf8-c46a9415975a)

**To Reproduce**
Document the code snipped below with doxygen. (default doxyfile)
```
class Test:
    ##
    # Default constructor of this class
    def __init__(self):
        ## var a
        self.a: int = 1

class SubTest(Test):
    def __init__(self):
        super().__init__()
        ## var b
        self.b: float = 0.5
        print(self.a)
        print(self.b)

if __name__ == "__main__":
    subTest = SubTest()
```

**Expected behavior**
The member "a" should only appear once and inherit its documentation from the Test class.

**Version**
OS: Win10
DoxyGen Version: 1.12.0

**Additional context**
I think this issue is somehow linked to issue #10973 / pull request  #10975

**Comment by doxygen** [comment]
@vniederb Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official release becomes available.

**Comment by vniederb** [comment]
I can confirm that this issue is no longer present. I used the latest commit to build the doxyGen binary. (2a62c197550990a6346d826cf0644a52d5e03e9b) 
Thanks for the fast replies and bug fixing.
I'm looking forward to the next release.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
